aquadoggo/src/test_utils/node.rs

```diff
@@ -161,7 +161,7 @@ pub async fn add_document(
         // We only want to issue dependency tasks.
         let dependency_tasks = tasks
             .iter()
-            .filter(|task| task.worker_name() == "depenedency");
+            .filter(|task| task.worker_name() == "dependency");
```

Comment:
😅

Reply:
Guess we never really wanted to issue dependency tasks either 😅